pysb/export/python.py

```diff
@@ -188,7 +187,7 @@ def ode_rhs(self, t, y, p):
                 ydot = self.ydot
                 weave.inline(r'''%s''', ['ydot', 't', 'y', 'p'])
                 return ydot
-            """, 8) % (pad('\n' + code_eqs, 16) + ' ' * 16))
+            """, 8) % ('\n' + pad(code_eqs, 16) + ' ' * 16))
```

Comment:
This just corrects some extraneous trailing whitespace on the first line of the output. I only noticed it because I re-generated robertson_standalone.py and git diff highlighted the trailing whitespace.